pkg/crc/machine/bundle/metadata_test.go

```diff
@@ -188,6 +190,11 @@ func TestGetBundleType(t *testing.T) {
 }
 
 func TestVerifiedHash(t *testing.T) {
+	// skip the test when running in disconnected env
+	if disconnected, _ := strconv.ParseBool(os.Getenv("CRC_IN_DISCONNECTED_ENV")); disconnected {
```

Comment:
I'd just use `LookupEnv` and would not try to parse it/... If it's set, ignore the test. But I'm fine with the way you wrote it.